app/controllers/application_controller.rb

```diff
@@ -35,4 +38,11 @@ def release_version
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: %i[email full_name])
   end
+
+  def check_privacy_policy_accepted
+    return if !current_user || current_user.admin? || current_user.privacy_policy_acceptance.present?
```

Comment:
Should be be showing this to lead provider users?

Reply:
That is an excellent question for Tom. I think we might have two privacy policies here.